include/z64.h

```diff
@@ -329,7 +329,11 @@ typedef struct {
     /* 0x010 */ s32 unk10;
     /* 0x014 */ UNK_TYPE1 pad14[0x2E];
     /* 0x042 */ s16 unk42;
-    /* 0x044 */ UNK_TYPE1 pad44[0x43];
+    /* 0x044 */ UNK_TYPE1 pad44[0x34];
+    /* 0x078 */ f32 previousPlayerHeight; //OvlDoorAna
```

Comment:
Looking at OOT, this is `respawn[RESPAWN_MODE_RETURN]`. I'd prefer if these were left as `unkX` for now, or for the entire respawn struct be carried over.